issuer-web/src/utils/index.ts

```diff
@@ -1,3 +1,38 @@
+import {BehaviorSubject} from 'rxjs'
+
+export function multiGuard(guards: any[]) {
```

Comment:
Curious about whether we could specify `guards: Function[]` here or the compiler would complain.